site/docs/install-windows.md

```diff
@@ -9,23 +9,52 @@ Supported Windows platforms:
 
 *   64 bit Windows 7 or higher, or equivalent Windows Server versions.
 
-Check
+_Check
 <a href="https://msdn.microsoft.com/en-us/library/windows/desktop/ms724832(v=vs.85).aspx">Microsoft's
-Operating System Version table</a> to see if your OS is supported.
+Operating System Version table</a> to see if your OS is supported._
 
-Before installing Bazel, make sure your system meets the system and software
-requirements. See the page about [using Bazel on Windows](windows.html).
+### 1. Install prerequisites (if not already installed)
 
-Install Bazel on Windows using one of the following methods:
+*   Python 2.7 or later.
+
+    Use the Windows-native Python version. Do not use Python that comes with the
+    MSYS2 shell or that you installed in MSYS using Pacman because it doesn't
+    work with Bazel.
+
+*   [MSYS2 shell](https://msys2.github.io/).
+
+    You also need to set the `BAZEL_SH` environment variable to point to
+    `bash.exe`. For example in the Windows Command Prompt (`cmd.exe`):
+
+    ```
+    set BAZEL_SH=C:\tools\msys64\usr\bin\bash.exe
+    ```
+
+    **Note**: do not use quotes (") around the path like you would on Unixes.
+    Windows doesn't need them and it may confuse Bazel.
+
+*   Several MSYS2 packages.
+
+    Run the following command in the MSYS2 shell to install them:
+
+    ```bash
+    pacman -Syuu git curl zip unzip
+    ```
+
+*   [Microsoft Visual C++ Redistributable for Visual Studio 2015](https://www.microsoft.com/en-us/download/details.aspx?id=48145)
+
+    This may already be installed on your system.
+
+### 2. Install Bazel on Windows using one of the following methods:
 
 *   [Download the binary (recommended)](#download-the-binary-recommended)
 *   [Install using Chocolatey](#install-using-chocolatey)
 *   [Compile Bazel from source](install-compile-source.html)
 
-## Download the binary (recommended)
+#### Download the binary (recommended)
 
 Go to Bazel's [GitHub releases page](https://github.com/bazelbuild/bazel/releases)
-and download the Windows binary.
+and download the Windows binary<sup>1</sup>: `bazel-<version>-installer-windows-x86_64.sh`.
```

Comment:
There's no installer on Windows. The user downloads a ready-to-run binary.
Please remove this information along with the footnote.

To expedite submitting this PR, I'll import it into the internal version control and make this edit myself, then send it to @spomorski .